ledger-core/virtual/integration/utils/server.go

```diff
@@ -174,11 +178,17 @@ func (s *Server) Init(ctx context.Context) {
 	if err := s.virtual.Init(ctx); err != nil {
 		panic(err)
 	}
-	s.Journal.Start()
 
 	s.pulseManager.AddDispatcher(s.virtual.FlowDispatcher)
 	s.IncrementPulseAndWaitIdle(ctx)
+}
+
+func (s *Server) StartRecording() {
+	s.StartRecordingExt(10_000, true)
```

Comment:
IMO discardOnOverflow should be false here, otherwise we'll silently timeout somewhere in the future

Reply:
all Replay* method will panic on overflow of the recording buffer. 
Discard here only clears memory, but doesn't remove overflow status.